framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts

```diff
@@ -212,6 +213,8 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 		let chainAccountJSON: ChainAccountJSON;
 		// Save blockHeader, aggregateCommit, validatorsData and cross chain messages if any.
 		try {
+			const nodeInfo = await this._sendingChainClient.node.getNodeInfo();
+			this._syncing = nodeInfo.syncing;
```

Comment:
if syncing is not used in any other place, we actually don't need to keep in in the property